src/main/java/core/basesyntax/Palindrome.java

```diff
@@ -21,6 +21,8 @@
      * <p>Результат: true</p>
      */
     public boolean isPalindrome(String text) {
-        return false;
+        StringBuilder result = new StringBuilder(text.replaceAll("[^\\w]", "").toLowerCase());
```

Comment:
let's create variable refactoredText with removed punctuations and all letter lowerCase
not to call these methods twice

Reply:
done